src/node_report_utils.cc

```diff
@@ -12,43 +12,51 @@ static constexpr auto null = JSONWriter::Null{};
 static void ReportEndpoint(uv_handle_t* h,
                            struct sockaddr* addr,
                            const char* name,
-                           JSONWriter* writer) {
+                           JSONWriter* writer,
+                           bool exclude_network) {
   if (addr == nullptr) {
     writer->json_keyvalue(name, null);
     return;
   }
 
   uv_getnameinfo_t endpoint;
   char* host = nullptr;
-  char hostbuf[INET6_ADDRSTRLEN];
   const int family = addr->sa_family;
   const int port = ntohs(family == AF_INET ?
                          reinterpret_cast<sockaddr_in*>(addr)->sin_port :
                          reinterpret_cast<sockaddr_in6*>(addr)->sin6_port);
 
-  if (uv_getnameinfo(h->loop, &endpoint, nullptr, addr, NI_NUMERICSERV) == 0) {
+  writer->json_objectstart(name);
+  if (!exclude_network &&
+      uv_getnameinfo(h->loop, &endpoint, nullptr, addr, NI_NUMERICSERV) == 0) {
     host = endpoint.host;
     DCHECK_EQ(port, std::stoi(endpoint.service));
+    writer->json_keyvalue("host", host);
+  }
+
+  if (family == AF_INET) {
+    char ipbuf[INET_ADDRSTRLEN];
+    if (uv_ip4_name(
+            reinterpret_cast<sockaddr_in*>(addr), ipbuf, sizeof(ipbuf)) == 0) {
+      writer->json_keyvalue("ip4", ipbuf);
```

Comment:
As new keys added, the [`NODE_REPORT_VERSION`](https://github.com/nodejs/node/blob/main/src/node_report.cc#L26) should be bumped.

Reply:
Should I maybe add a TCP schema in the docs of getReport, right now there isn't any https://github.com/nodejs/node/blob/main/doc/api/report.md?plain=1#L241-L326